src/Block/block.php

```diff
@@ -25,8 +27,29 @@ function () use ( $block_json_path ) {
 	);
 }
 
-function render() {
-	$iframe_url = plugins_url() . '/chatrix/build/block/app.html';
+function render(): string {
+	$login_token = null;
+	if ( isset( $_GET['loginToken'] ) ) {
+		// phpcs:ignore
+		$login_token = $_GET['loginToken'];
```

Comment:
Rather than ignoring we can use [sanitize_text_field](http://developer.wordpress.org/reference/functions/sanitize_text_field/) like this:
```suggestion
		$login_token = sanitize_text_field( $_GET['loginToken'] );
```